sympy/printing/pretty/pretty.py

```diff
@@ -1017,6 +1030,10 @@ def _print_Feedback(self, expr):
 
         return self._print(num)/self._print(den)
 
+    def _print_TransferFunctionMatrix(self, expr):
+        from sympy.matrices import Matrix
+        return self._print_Matrix(Matrix(expr.args[0]))
```

Comment:
The error log shows _print_Matrix is not a PrettyPrinter attribute.
Might be `_print_MatrixBase`  https://github.com/sympy/sympy/blob/516b0b2bdab790f5b9b402a88b90f50c2af10d8c/sympy/printing/pretty/pretty.py#L763

Reply:
Yep, I removed this attribute recently

Reply:
This should be just
```suggestion
        return self._print(Matrix(expr.args[0]))
```